extra/program-addresses.md

```diff
@@ -0,0 +1,17 @@
+---
+description: A list of Dual Finance Solana program addresses
+---
+
+# Program Addresses
+
+|Dual Finance Program| Solana Program Address|
+|---| ---|
+|DIP Smart Contract| DiPbvUUJkDhV9jFtQsDFnMEMRJyjW5iS6NMwoySiW8ki|
+|SO Smart Contract | 4yx1NJ4Vqf2zT1oVLk4SySBhhDJXmXFt88ncm4gPxtL7|
+|DIP MM Smart Contract| esdUULhoeFHro46mvkTizMzg38hKBunvErmPuUw6n9p|
```

Comment:
I wouldnt include this one since it is not really user facing

Reply:
Removed.